rails-kickoff-template.rb

```diff
@@ -113,24 +118,22 @@ def setup_bullet
 end
 
 def output_final_instructions
-  after_bundle do
-    msg = <<~MSG
-      Template Completed!
-
-      Please review the above output for issues.
-
-      To finish setup, you must prepare Heroku with at minimum the following steps (review the developer guide for further details)
-      1) Setup the Skylight ENV variable
-      2) Configure Sentry
-      3) Add the jemalloc buildpack:
-        $ heroku buildpacks:add --index 1 https://github.com/gaffenyc/heroku-buildpack-jemalloc.git
-      4) Setup Redis (if using Sidekiq)
-      5) Review your README.md file for needed updates
-      6) Review your Gemfile for formatting
-    MSG
-
-    say msg, :magenta
-  end
+  msg = <<~MSG
+    Template Completed!
+
+    Please review the above output for issues.
+
+    To finish setup, you must prepare Heroku with at minimum the following steps (review the developer guide for further details)
+    1) Setup the Skylight ENV variable
+    2) Configure Sentry
+    3) Add the jemalloc buildpack:
+      $ heroku buildpacks:add --index 1 https://github.com/gaffenyc/heroku-buildpack-jemalloc.git
+    4) Setup Redis (if using Sidekiq)
+    5) Review your README.md file for needed updates
+    6) Review your Gemfile for formatting
+  MSG
+
+  say msg, :magenta
```

Comment:
any reason you removed this from after_bundle? won't this now run before bundle completes and you won't see the message without scrolling up?